ajax/libs/tui.calender/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "tui.calender",
```

Comment:
Please fetch name according to https://github.com/nhnent/tui.calendar/blob/2a1d526f9491855dc83131334bc5fbf6f9c80560/package.json#L2

Reply:
Hi @sashberd, thanks for the review, but the name of repository  is tui.calender and in the docs its specifies that name in package.json should match with repository name. Please suggest what to do
Thanks

Reply:
Well in package.json of lib the name is : tui-calendar

Reply:
I also had the same confusion hence I tried it using tui-calender ,but it failed in CI .You can see the report https://ci.cdnjs.com/cdnjs/cdnjs/21784

Also please suggest what keyword needs to be added
Thanks for help